pkgs/development/python-modules/flask-mysqldb/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, flask
+, mysqlclient
+}:
+
+buildPythonPackage rec {
+  pname = "flask-mysqldb";
+  version = "1.0.1";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "alexferl";
+    repo = "flask-mysqldb";
+    rev = "v${version}";
+    hash = "sha256-qHPepvNA6oK2ZYAl9NqWepqwFSWOGu2KWuOJOe9V05g=";
+  };
+
+  propagatedBuildInputs = [
+    flask
+    mysqlclient
+  ];
+
+  pythonImportsCheck = [
+    "flask_mysqldb"
+  ];
+
```

Module has no unittests.

```suggestion

  # Module has no tests
  doCheck = false;
```

---

Fixed.